docs/porting-guide.rst

```diff
@@ -2445,9 +2445,12 @@ of the CPU to enable quick crash analysis and debugging. It requires that a
 console is designated as the crash console by the platform which will be used to
 print the register dump.
 
-The following functions must be implemented by the platform if it wants crash
-reporting mechanism in BL31. The functions are implemented in assembly so that
-they can be invoked without a C Runtime stack.
+By default, the definitions in `plat/common/aarch64/platform\_helpers.S` will
+cause the crash output to be routed over the same console that was registered
+for boot messages. If necessary, platforms may supply their own definitions of
```

Comment:
If you can guarantee that early crashes on platforms that don't override these functions, results in nothing being output and the plat_panic_handler() being correctly called, then this should be OK. However, this seems to be separate to the main part of this patch and could come in a later patch. The additional comments you suggest would help too.